app/controller/window/ParallelLineWindow.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * This class is the controller for the ParallelLine.
+ */
+Ext.define('CpsiMapview.controller.window.ParallelLine', {
+    extend: 'Ext.app.ViewController',
+
+    alias: 'controller.cmv_parallel_line_window',
+
+    requires: [
+        'BasiGX.view.component.Map'
+    ],
+
+    selectInteraction: null,
+
+    layer: null,
+
+    onSelectFeatureToggle: function (btn, toggled) {
+        var me = this;
+        var view = me.getView();
+        var vm = view.getViewModel();
+        var map = BasiGX.view.component.Map.guess().getMap();
+        if (toggled) {
+            var interactionConfig = view.interactionConfig ? view.interactionConfig : {};
+            this.selectInteraction = new ol.interaction.Select(interactionConfig);
+            this.selectInteraction.on('select', function (evt) {
+                var selectedFeature = evt.selected[0];
```

Comment:
is it possible that the user accidentally selects two features? this could maybe be handled

Reply:
As described in the PR description, `ParallelLineWindow` is just an example implementation and not intended for production use. The actual component is `PrallelLine` toolbar.

Reply:
alright